de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537900_sys_gh5856_apparently_the_sysconfig_was_never_retrieved_cause_of_org_and_i_lost_40_minutes_on_this.sql

```diff
@@ -0,0 +1 @@
+update AD_SysConfig set ad_client_id = 0 WHERE Name='de.metas.payment.autoAssignToSalesOrderByExternalOrderId.enabled';
```

Comment:
right. so, the setting cna be on *System* level, but with `ConfigurationLevel=O`, one can later override it on *Org* level (also on * Client* level)